test/unit-test/cellular_3gpp_urc_handler_utest.c

```diff
@@ -427,6 +501,72 @@ void test_Cellular_ParseRegStatus_Regs_OutOfRange( void )
     TEST_ASSERT_EQUAL( CELLULAR_PKT_STATUS_FAILURE, packetStatus );
 }
 
+/**
+ * @brief Test that register out of lower bound range case _Cellular_ParseRegStatus to return CELLULAR_PKT_STATUS_FAILURE.
+ */
+void test_Cellular_ParseRegStatus_Regs_OutOfLowerBoundRange( void )
+{
+    CellularPktStatus_t packetStatus = CELLULAR_PKT_STATUS_OK;
+    CellularContext_t context;

```

Comment:
The context is not initialized. Suggest to initialized to { 0 } to prevent random result and also suggest to check local variable in other functions with initial value.

Reply:
Thanks for the suggestion.